src/quests/level10.ts

```diff
@@ -0,0 +1,45 @@
+import { familiarWeight, use, useFamiliar, weightAdjustment } from "kolmafia";
+import {
+  $familiar,
+  $item,
+  $location,
+  $skill,
+  adventureMacro,
+  get,
+  have,
+  maximizeCached,
+  Mood,
+  set,
+} from "libram";
+
+import { Macro } from "../combat";
+import { questStep } from "../lib";
+import { addFamiliarWeight, moodNoncombat } from "../mood";
+
+export function airship(): void {
+  if (questStep("questL10Garbage") < 7) {
+    if (questStep("questL10Garbage") < 1) use(1, $item`enchanted bean`);
+
+    const freeRunFamiliar = have($familiar`Frumious Bandersnatch`)
+      ? $familiar`Frumious Bandersnatch`
+      : $familiar`Pair of Stomping Boots`;
+    if (!get<boolean>("_bcas_banderRunawaysUsed") && have(freeRunFamiliar)) {
+      useFamiliar(freeRunFamiliar);
+      addFamiliarWeight(moodNoncombat()).execute();
+      maximizeCached(["Familiar Weight", "-Combat Rate"]);
```

Comment:
Do we want to weigh familiar weight more as we start to run out of runs?

Reply:
I guess we aren't really worried about that happening, but if we'd like to eke out more lbs, we can do that

Reply:
yeah, we should, but not for this PR